src/components/molecules/TempChart.tsx

```diff
@@ -0,0 +1,118 @@
+import { scaleLinear, scaleTime } from '@visx/scale';
+import { GridRows, GridColumns } from '@visx/grid';
+import { Group } from '@visx/group';
+import { AxisLeft, AxisBottom } from '@visx/axis';
+import { curveBasis } from '@visx/curve';
+import { LinePath } from '@visx/shape';
+import { Threshold } from '@visx/threshold';
+
+import React from 'react';
+
+interface Data {
```

Comment:
```suggestion
interface IData {
```

Interaces start with `I`.